Alicerce/Sources/Logging/Log+NodeLogDestination.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  Log+NodeLogDestination.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public extension Log {
+
+    public class NodeLogDestination : LogDestination, LogDestinationFallible {
+
+        enum Error: Swift.Error {
+            case httpError(statusCode: Int)
+            case network(Swift.Error)
+        }
+
+        private static let dispatchQueueLabel = "com.mindera.alicerce.log.destination.node"
+        private static let defaultRequestTimeout: TimeInterval = 0
```

Comment:
The default value on a `URLSessionConfiguration.timeoutIntervalForRequest` is `60`, so I think `0` is a probably a bit too harsh as a default 😆